src/modules/reporting/components/reporting-dispute-form/reporting-dispute-form.jsx

```diff
@@ -456,10 +456,19 @@ export default class ReportingDisputeForm extends Component {
         <li>
           <div className={Styles.ReportingDisputeForm__outcome_selection_msg}>
             Please choose an outcome below based on the resolution source at the
-            time the market ended. The market should be considered invalid if it
-            is opinion-based, unclear, has more than one valid outcome, or the
-            outcome wasn&#39;t known when the market ended.
+            time the market ended.
```

Comment:
Small change at end of statement:

Please choose an outcome based on the the resolution source when the market ends.